### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by Maurice-MSDA** [hammem/monarchmoney#107] [General]
Hello, I am getting a 404 error when trying to login. Any Ideas why?

**Comment by hammem** [comment]
@Maurice-MSDA , if you can share an example of the code you are trying to run, as well as the exception stack trace, we may be able to determine the cause of the issue.

**Reply by Maurice-MSDA** [reply]
@hammem thanks for your quick response. I was able to resolve my issue. I used the wrong token.
